### PR TITLE
Testing branch

### PR DESCRIPTION
**Pull request by VickTor61**
made the new changes

### CODE REVIEW

File: app/controllers/campaigns_controller.rb [hunk 1]
```diff
@@ -1,54 +1,53 @@
 class CampaignsController < ApplicationController
-  # before_action :authenticate_user!
+  before_action :authenticate_user!
 
- def index
-    @campaigns = Campaign.all
- end
+  def index
+    @campaigns = current_user.campaigns.all
```
Comment (EnomaDebby) [inline app/controllers/campaigns_controller.rb]: Remove the .all here

File: config/database.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 development:
   adapter: postgresql
   encoding: unicode
-  database: database_name_development
+  database: postgres_development
```
Comment (EnomaDebby) [inline config/database.yml]: This should be the name of your app not postgres. Example sales_app_development

**Review by EnomaDebby — COMMENTED** [review]
Good job 👏. Make the changes and push and I’ll approve

File: app/controllers/campaigns_controller.rb (hunk 1 above)
Comment (VickTor61) [reply app/controllers/campaigns_controller.rb]: I already made that correction in the testing branch

Comment (EnomaDebby) [reply app/controllers/campaigns_controller.rb]: Is it not the .all I’m still seeing here? Just current_user.campaigns is enough

Comment (VickTor61) [reply app/controllers/campaigns_controller.rb]: oh i thought u meant the Campaign.all should be change to current_user.campaigns.all. working on it now asap

**Review by EnomaDebby — APPROVED** [review]

**Pull request merged**